server/.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit =
+    tests/**
+    **/__init__.py
+    **/migrations/*
+    **/monkey_patch_for_testing.py  
```

Comment:
Maybe ignore `./manage.py`, `server/wsgi.py`, and `server/asgi.py`, but those are very minor. I've seen some other django projects ignore `*/scripts/*`, but I think that's arguable.


This should be a follow up PR/issue for later (unless you look in to it and it is trivial to set up), but looking at other project's `.coveragerc` files reminded me that there's a coverage plugin called `django_coverage_plugin` we can add to possibly get coverage reports on django templates as well. Note, the example I saw this in was using django's built in templates, while this project is using jinja for templating. Haven't checked if this specific plugin can also track coverage for jinja files, or if some jinja alternative exists.
```
plugins =
    # allows coverage for templates
    django_coverage_plugin
```